### DISCUSSION TITLE
Smoothie website status

### DISCUSSION THREAD

**Discussion by thomaskilian** [luc-github/ESP3D#641] [Questions]
Maybe not the right place, but I don't know any other. The website is down the 2nd day. Any idea what's going on?

**Comment by luc-github** [comment]
did you asked on smoothieware repository?

**Comment by thomaskilian** [comment]
I was starting from https://github.com/luc-github/Smoothieware Sorry, if I got it wrong. Unfortunately the Smoothiware page on github does not have a discussion :-/

**Reply by luc-github** [reply]
I am not related to smoothieware 
but you can open issue on smoothieware github to ask your question : https://github.com/Smoothieware/Smoothieware

**Reply by thomaskilian** [reply]
I will rather do that. Sorry for disturbing!

**Reply by luc-github** [reply]
no problem it is discussion place - hope you get answer

**Reply by thomaskilian** [reply]
They fixed it right after the report :-)
